contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol

```diff
@@ -27,19 +29,24 @@ struct Upkeep {
   uint32 executeGas;
   uint32 maxValidBlocknumber;
   bool paused;
-  address target;
+  AutomationForwarder forwarder;
   // 3 bytes left in 1st EVM word - not written to in transmit
   uint96 amountSpent;
   uint96 balance;
   uint32 lastPerformBlockNumber;
   // 4 bytes left in 2nd EVM word - written in transmit path
+  address target;
```

Comment:
do we need to know this too if we know the forwarder?

Reply:
used in checkUpkeep. We could route that through forwarder too, but I'm not sure if it's really simpler?

Reply:
I see, yeah seems easier to store it but we'll need to gas optimize this so that it's not read during performUpkeep path.

Reply:
yeah that's why it's last in the struct